src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -531,6 +531,9 @@ void VtolAttitudeControl::task_main()
 			// update transition state if got any new data
 			if (got_new_data) {
 				_vtol_type->update_transition_state();
+
+				_vtol_type->process_mc_data();
+				fill_mc_att_rates_sp();
```

Comment:
I haven't fixed it yet, as I couldn't pinpoint the location yet. If you can log-munch one of your test logs it should be obvious if this still persists.